apps/web/components/ProposalCard.tsx

```diff
@@ -205,33 +205,25 @@ export function ProposalCard({
         <div className="flex gap-12 flex-wrap w-full ">
           <div className="mt-4 w-full">
             {isAllocationView ?
-              <div className=" flex w-full flex-wrap items-center justify-between gap-6">
+              <div className="flex w-full flex-wrap items-center justify-between gap-6">
                 <div className="flex items-center gap-8 flex-grow flex-wrap">
                   <div className="flex-grow sm:max-w-[460px]">
-                    <div
-                      className={isProposalEnded ? "tooltip" : ""}
-                      data-tip={
-                        isProposalEnded ?
-                          "Cannot change allocation when a proposal is ended"
-                        : ""
-                      }
-                    >
-                      <input
-                        type="range"
-                        min={0}
-                        max={Number(memberActivatedPoints)}
-                        value={inputData ? Number(inputData.value) : undefined}
-                        className={`range range-md cursor-pointer bg-neutral-soft [--range-shdw:var(--color-green-500)] ${isProposalEnded ? "grayscale !cursor-not-allowed" : ""}`}
-                        step={Number(memberActivatedPoints / 100n)}
-                        onChange={(e) => {
-                          inputHandler(
-                            proposalData.id,
-                            BigInt(Math.floor(Number(e.target.value))),
-                          );
-                        }}
-                        disabled={isProposalEnded}
-                      />
-                    </div>
+                    <input
```

Comment:
Missing the tooltip logic

Reply:
I remove it because it think was not either neccesary that tooltip